CatAnalyzer/scripts/catGetDatasetInfo

```diff
@@ -44,6 +44,9 @@ nevtIdx = csv[0].index("NEvent")
 lumiIdx = csv[0].index("luminosity (pb-1)")
 colourIdx = csv[0].index("colour")
 pathIdx = csv[0].index("Path")
+DataSetNameIdx = csv[0].index("DataSetName")
+GlobalTagIdx = csv[0].index("GlobalTag")
+LumiMaskIdx = csv[0].index("LumiMask")
```

Comment:
I thought it would be easier to keep all dataset info in one place.
and have crab submission read from that too.